### ISSUE TITLE
No support for multi-image .tif

### ISSUE THREAD

**Issue by 0oel** [opencv/cvat#5174]
### My actions before raising this issue
- [X] Read/searched [the docs](https://github.com/cvat-ai/cvat/tree/master#documentation)
- [ X] Searched [past issues](/issues)

I'm trying to annotate multi-image `.tif` files. While the files will upload succesfully, CVAT will only show the first frame (whereas the image has 50+). 

### Expected Behaviour
You should be able to upload a multi-image `.tif` and it should upload all the frames, like CVAT does with video files. 

### Steps to Reproduce (for bugs)
1. Upload a multi image `.tif`. 
2. It only let's you annotate the 1st frame

### Your Environment
- Docker version `docker version` (e.g. 20.10.18, build b40c2f6):
- Operating System and version (e.g. Ubuntu 22.04.1 LTS):

**Comment by 0oel** [comment]
I do think `.tif` support must have been added recently.

**Comment by HStuart18** [comment]
I don't see this functionality but agree that it would be immensely useful.

**Comment by 0oel** [comment]
Workaround so far. If you have a multi image `.tif`, you can use Image Magick to convert it. 
On MacOS:
`brew install imagemagick` 
Then 
`convert image.tif image%d.tif`, which splits up the file into it's frames starting at index 0.

**Comment by harrystuart** [comment]
Yes you can do so. If you then select your step size to be the number of pages per tiff, then you can quickly skip across tiffs while viewing the same channel. However, the caching will only apply to adjacent frames, which are likely adjacent channels. Therefore, a more integrated solution is desirable.
